vhdbuilder/release-notes/AKSWindows/2019/17763.1935.210513.txt

```diff
@@ -0,0 +1,148 @@
+c:\akse-cache\win-k8s\v1.18.14-hotfix.20210511-1int.zip                          F36E2605C3E6EDA8ABE0394AF22CF0BB4A383B1FEE35041A1C79D0C596D99C25  99501775
+c:\akse-cache\win-k8s\v1.18.17-hotfix.20210322-1int.zip                          75A33B623827EF9745DB23094074A65060B9DB39D0DC0DCCC8F09360FA4CBC55  58090328
+c:\akse-cache\win-k8s\v1.18.17-hotfix.20210505-1int.zip                          E463F6B3D300BD80091A86EB4A3A462C712E03A09B99B65FA55C30348E1C1223  58072067
+c:\akse-cache\win-k8s\v1.18.18-hotfix.20210504-1int.zip                          5C6F696CFB2F848C62D7A618231C4D911BFD407509F291616E3552CDA0D0CC80  58134168
+c:\akse-cache\win-k8s\v1.19.10-hotfix.20210504-1int.zip                          7C9A1F490AEEBE56ABC8B3266FD3CA5F36B3FB0BB4423E47C9F9D0C5FE3A477C  56486491
+c:\akse-cache\win-k8s\v1.19.6-hotfix.20210118-1int.zip                           B0D6258B0AD8456EEF07B573E92601A2597BDCA7932371049387152077B3AB46 102435304
+c:\akse-cache\win-k8s\v1.19.7-hotfix.20210428-1int.zip                           233A05611BE22E1356DD4BC435741D0025D073A17560693444F70786F4B4F63B  56436003
+c:\akse-cache\win-k8s\v1.19.7-hotfix.20210511-1int.zip                           C18B8C7000D3B5D767DF94843D8AF8B1916C4875557B799991BF2E2331FA976B  56436153
+c:\akse-cache\win-k8s\v1.19.9-hotfix.20210322-1int.zip                           C1E5021CFBC9CCC66B52D9E3087D2B17F3A0FFCEA0CE91C98180867F2B522758  56500356
+c:\akse-cache\win-k8s\v1.19.9-hotfix.20210505-1int.zip                           E1826314CFFBDD73942E3D0A04C3081A155DD57F8464AD6BA253EABF1C3C4D37  56468022
+c:\akse-cache\win-k8s\v1.20.2-hotfix.20210428-1int.zip                           4459AE9DF44CF433CFA77C1B67003988D5DFBF7180AF309A7A610E212C0A103D  56971327
+c:\akse-cache\win-k8s\v1.20.2-hotfix.20210511-1int.zip                           5B7B3A311155C8C883FB36AE1616CA5399590F97F3F8C755B8781F6F5F14A055  56971504
+c:\akse-cache\win-k8s\v1.20.5-hotfix.20210322-1int.zip                           9B502F037A2F3AD6AD86475EDF5F5BC10885A7E3DF8CA5C715BCBE5F7041E9BA  57030477
+c:\akse-cache\win-k8s\v1.20.5-hotfix.20210505-1int.zip                           A2439F996C2B4FE95905640C02A3C6047D9C3FBF2ECA6BD6C9998D81F776947D  57004010
+c:\akse-cache\win-k8s\v1.20.6-hotfix.20210504-1int.zip                           E849906B54EDE8D01CBF81E33BDAB432CBC69032938D7BA72EDC76ED34D4A0F2  57015331
+c:\akse-cache\win-k8s\v1.21.0-1int.zip                                           28D24B1F807DF1DEBB6DFC518FF19EB067E1166942452938A100463D2EAF1542  59134044
+c:\akse-cache\win-vnet-cni\azure-vnet-cni-singletenancy-windows-amd64-v1.2.2.zip 52306F546DCB87D266A6DFDBD545982E6292134DEB86B8F90E00485ACFD6B4A1  39555626
+c:\akse-cache\win-vnet-cni\azure-vnet-cni-singletenancy-windows-amd64-v1.2.6.zip B2FF69608D97B963F8B0B0631F9E88329B006D954773566F019680DED75B8774  72629816
+c:\akse-cache\win-vnet-cni\azure-vnet-cni-singletenancy-windows-amd64-v1.2.7.zip 9AB8FA4C3FBD31429F3D6E5F8D9CE2FF1B192F52F43C5FA1ABA97FEC9BEE9927  72631755
+

```

Comment:
why 4 empty lines?

Reply:
These files are generated automatically. We can consider to remove these empty lines later.